common/djangoapps/util/date_utils.py

```diff
@@ -62,6 +63,19 @@ def get_time_display(dtime, format_string=None, coerce_tz=None):
         return get_default_time_display(dtime)
 
 
+def get_formatted_time_zone(time_zone, option=None):
+    """
+    Returns a formatted time zone (e.g. 'Asia/Tokyo (JST +0900)') for user account settings time zone drop down
+    """
+    abbr = now().astimezone(timezone(time_zone)).strftime('%Z')
+    offset = now().astimezone(timezone(time_zone)).strftime('%z')
+    if option == 'abbr':
+        return abbr
+    elif option == 'offset':
+        return offset
+    return "{name} ({abbr}, UTC{offset})".format(name=time_zone, abbr=abbr, offset=offset).replace("_", " ")
```

Comment:
Can't remember who, but someone said they'd prefer not seeing the `_`s. So `America/New_York (EDT, UTC-400)` would be `America/New York (EDT, UTC-400)`